.github/workflows/ci.yml

```diff
@@ -29,27 +36,22 @@ jobs:
     - name: Install npm dependencies
       run: npm ci
 
-    - name: Install python dependencies
-      run: python3 -m pip install -r requirements.txt
+    - name: Install pip dependencies
+      run: |
+        pip install --upgrade pip
```

Comment:
This currently doesn't do anything, and I don't think we'll need it into the future.

Reply:
Ok, I removed `pip` upgrade.